packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -481,7 +446,7 @@ class ScrollableBase extends Component {
 
 			if (isPageUp(keyCode) || isPageDown(keyCode)) {
```

Comment:
What about putting `(direction === 'vertical' || direction === 'both)` here?

Reply:
I added it properly.